openpype/lib/avalon_context.py

```diff
@@ -1971,3 +1971,123 @@ def get_last_workfile(
         return os.path.normpath(os.path.join(workdir, filename))
 
     return filename
+
+
+@with_avalon
+def get_linked_ids_for_representations(project, repre_ids, dbcon=None,
+                                       link_type=None, max_depth=0):
```

Comment:
Just more specific arg name
```suggestion
def get_linked_ids_for_representations(project_name, repre_ids, dbcon=None,
                                       link_type=None, max_depth=0):
```